lighthouse-treemap/app/src/main.js

```diff
@@ -734,13 +734,41 @@ function showError(message) {
   document.body.textContent = message;
 }
 
+/**
+ * @param {string} encoded
+ */
+function fromBinary(encoded) {
+  const binary = atob(encoded);
+  const bytes = new Uint8Array(binary.length);
+  for (let i = 0; i < bytes.length; i++) {
+    bytes[i] = binary.charCodeAt(i);
+  }
+  return String.fromCharCode(...new Uint16Array(bytes.buffer));
+}
+
 async function main() {
+  /** @type {Record<string, any>} */
+  let params = {};
+  if (Object.fromEntries) {
```

Comment:
i figured i should do the bare minimum so that the post message stuff at least works in ie.

Reply:
> i figured i should do the bare minimum so that the post message stuff at least works in ie11. prob not needed.

unless we run it through babel, the treemap bundle isn't going to get past parsing in IE11 (spread/rest, default params, probably others I'm not thinking of), so there's not much to be done to be defensive in the code itself